transport/src/main/java/io/netty/channel/socket/DatagramChannelConfig.java

```diff
@@ -180,4 +181,16 @@
 
     @Override
     DatagramChannelConfig setMessageSizeEstimator(MessageSizeEstimator estimator);
+
+    @Override
+    @Deprecated
+    DatagramChannelConfig setWriteBufferHighWaterMark(int writeBufferHighWaterMark);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)